src/templates/components/CommunityTemplateInstallOverlay.tsx

```diff
@@ -122,6 +122,7 @@ class CommunityTemplateInstallOverlayUnconnected extends PureComponent<Props> {
         this.props.resourcesToSkip,
         this.props.stagedTemplateEnvReferences
       )
+      event('template_install', {templateID: this.props.org.id})
```

Comment:
this code is calling the org id the `templateId`. I think the url might be appropriate: 

```ts
event('template_install', {templateUrl: this.props.stagedTemplateUrl})
```